### PR TITLE
headset initially mispositioned

### PR DESCRIPTION
**Pull request by pandafriggo**
took the opportunity to remove some logging that clutter the console

### CODE REVIEW

File: crates/bevy_webxr/src/interaction/input.rs [hunk 1]
```diff
@@ -70,7 +70,7 @@ pub fn handle_input(action_set: &mut Mut<XrActionSet>, frame: &web_sys::XrFrame)
             let button = match button {
                 Some(b) => b,
                 None => {
-                    warn!("Button does not exist");
+                    // warn!("Button does not exist");
```
Comment (casey-celestin-mk) [inline crates/bevy_webxr/src/interaction/input.rs]: Please just remove

File: crates/bevy_webxr/src/interaction/input.rs [hunk 2]
```diff
@@ -90,9 +90,9 @@ pub fn handle_input(action_set: &mut Mut<XrActionSet>, frame: &web_sys::XrFrame)
                 .map(|v| v.as_bool())
                 .unwrap_or(None);
 
-            if button == "xr-standard-trigger" && handedness_string == "right" {
-                info!("{:?}", value.clone());
-            }
+            // if button == "xr-standard-trigger" && handedness_string == "right" {
+                // info!("{:?}", value.clone());
```
Comment (casey-celestin-mk) [inline crates/bevy_webxr/src/interaction/input.rs]: Please remove

**Review by casey-celestin-mk — CHANGES_REQUESTED** [review]

File: crates/bevy_webxr/src/lib.rs [hunk 3]
```diff
@@ -172,23 +170,33 @@ fn setup_webxr_pawn(
...
     let left_proj: Mat4 = left_eye.projection_matrix().xr_into();
 
     let fov = fov_from_mat4(left_proj);
 
     info!("fov:{:?} deg", fov.to_degrees());
 
-    let left_tf: Transform = left_eye.transform().xr_into();
+    // we'd previously get this transform from left_eye XrView, but the world origin must be fixed
+    // regardless of user position/orientation
+    let left_tf = Transform {
```
Comment (casey-celestin-mk) [inline crates/bevy_webxr/src/lib.rs]: `Transform::IDENTITY` would be cleaner

File: crates/bevy_webxr/src/lib.rs [hunk 4]
```diff
@@ -172,23 +170,33 @@ fn setup_webxr_pawn(
...
     let right_eye: &web_sys::XrView = views
         .iter()
         .find(|view| view.eye() == web_sys::XrEye::Right)
         .unwrap();
-    // let right_eye_mat = right_eye.projection_matrix().xr_into();
 
-    let right_tf: Transform = right_eye.transform().xr_into();
+    // we'd previously get this transform from right_eye XrView, but the world origin must be fixed
+    // regardless of user position/orientation
+    let right_tf = Transform {
```
Comment (casey-celestin-mk) [inline crates/bevy_webxr/src/lib.rs]: `Transform::IDENTITY` would be cleaner

**Review by casey-celestin-mk — CHANGES_REQUESTED** [review]

File: crates/bevy_webxr/src/lib.rs (hunk 3 above)
Comment (pandafriggo) [reply crates/bevy_webxr/src/lib.rs]: done

File: crates/bevy_webxr/src/lib.rs (hunk 4 above)
Comment (pandafriggo) [reply crates/bevy_webxr/src/lib.rs]: done

File: crates/bevy_webxr/src/interaction/input.rs (hunk 2 above)
Comment (pandafriggo) [reply crates/bevy_webxr/src/interaction/input.rs]: done

File: crates/bevy_webxr/src/interaction/input.rs (hunk 1 above)
Comment (pandafriggo) [reply crates/bevy_webxr/src/interaction/input.rs]: done

**Review by casey-celestin-mk — CHANGES_REQUESTED** [review]
Fixes rotation issues! Great job

**Review by casey-celestin-mk — APPROVED** [review]
Fixes rotation issues! Great job

**Pull request merged**